testing/unit-tests.php

```diff
@@ -32,14 +23,14 @@
 require 'unit-tests/_files.php';
 
 // Setup test files
-$test = new GroupTest('CSSTidy unit tests');
+$test = new TestSuite('CSSTidy unit tests');
```

Comment:
This class' name was changed: https://github.com/simpletest/simpletest/blob/v1.1.7/HELP_MY_TESTS_DONT_WORK_ANYMORE#L53